platform/admin/ui/src/main/webapp/templates/installer/navigationButtons.handlebars

```diff
@@ -42,20 +42,12 @@
     {{/ifNotAnd}}
 {{else}}
     <li>
-        {{#if modified}}
-            {{#if busy}}
-                <a id="shutdownLaterStep" class="disabled-next" href="#">Restart Later</a>
-                <a id="shutdownNowStep" class="disabled-next" href="#">Restart Now</a>
-            {{else}}
-                <a id="shutdownLaterStep" class="next shutdown-later" href="#">Restart Later</a>
-                <a id="shutdownNowStep" class="next shutdown-now" href="#">Restart Now</a>
-            {{/if}}
+        {{#if busy}}
+            <a id="shutdownLaterStep" class="disabled-next" href="#">Shutdown Now</a>
+            <a id="shutdownNowStep" class="disabled-next" href="#">Restart Now</a>
         {{else}}
-            {{#if busy}}
-                <a id="finishStep" class="disabled-next" href="#">Finish</a>
-            {{else}}
-                <a id="finishStep" class="next finish" href="#">Finish</a>
-            {{/if}}
+            <a id="shutdownLaterStep" class="next shutdown-later" href="#">Shutdown Now</a>
```

Comment:
❗️ The `id` and `class` for these two elements don't match with the actual text, which may cause confusion and mislead people making changes to the css, JavaScript or tests in the future. We should consider using `id="shutdownNowStep" class="next shutdown-now"` and `id="restartNowStep" class="next restart-now"` respectively.

Reply:
I'm in agreement here, thanks for catching this!